packages/google_maps_flutter/lib/src/pattern_item.dart

```diff
@@ -0,0 +1,24 @@
+part of google_maps_flutter;
+
+/// Immutable item used in the stroke pattern for a Polyline.
+class PatternItem {
+  const PatternItem._(this._json);
+
+  static const PatternItem dot = PatternItem._(<dynamic>['dot']);
+
+  /// [length] non-negative.
```

Comment:
this is not a really useful comment. Let's change this to:

```
/// A dash used in the stroke pattern for a [Polyline].
///
/// [length] has to be non-negative.
```

please fix this for `gap` as well.